generated/niswitch/niswitch.proto

```diff
@@ -7,7 +7,7 @@
 syntax = "proto3";
 
 option java_multiple_files = true;
-option java_package = "com.ni.grpc.niswitch";
+option java_package = "com.ni.grpc.switch";
```

Comment:
We'd have to tag these as `breaking_change`. But I expect it's OK to change these. Would be an "easy-to-fix source-breaking change" affecting a small group of clients (possibly none).

Reply:
Verified it's fixed now (back to what it used to be).